rest_framework/templates/rest_framework/admin.html

```diff
@@ -230,6 +230,9 @@ <h4 class="modal-title" id="myModalLabel">{{ error_title }}</h4>
 {% if filter_form %}{{ filter_form }}{% endif %}
 
         {% block script %}
+            <script>
+              var csrf_cookie_name = "{{ csrf_cookie_name|default:'csrftoken' }}";
```

Comment:
Could we instead do something like:

```javascript
window.drf = {
  csrfCookieName: "{{ csrf_cookie_name|default:'csrftoken' }}"
};
```
and use `window.drf.csrfCookieName` in `csrf.js`?